doc/theoretical_description_regression.rst

```diff
@@ -216,4 +248,8 @@ References
 ==========
```

Comment:
Could you please :
- add a bullet point on J+aB in the "Key takeaways" section, mentionning its improved sensitivity to epistemic uncertainty (cf example) ?
- Update the comparative table, introducing B the number of bootstraps in the text just above ?

Reply:
J'ai aussi modifié les autres points pour y mentionner J-aB
je pense que dans (cf example) tu as en tête le notebook que j'avais fait avec les distributions de points verts montrant les différentes prédictions des différents estimateurs ré-entraînés.  Mais je ne l'ai qu'en local sur une branche, et il n'est pas dans les exemples "officiels", car il est fondée sur une version "augmentée" de MAPIE qui a comme attribut la matrice des prédictions des estimateurs sur l'ensemble test
Donc je n'ai pas mis la ref "cf example"

J'attends de pouvoir générer la doc en local et je fais le push (demain car le débit wifi est bridé)